### ISSUE TITLE
training with one GPU

### ISSUE THREAD

**Issue by wo0423** [charlesCXK/TorchSemiSeg#73]
excuse me,i want to know how to train with only one GPU, can you help me? thanks a lot.

**Comment by charlesCXK** [comment]
Hi, we highly recommend using 4 or 8 GPUs to train the model, since the performance is very sensitive to the batch size.

To train with one GPU, you could set a smaller batch size and linearly reduce the learning rate.

**Comment by wo0423** [comment]
Hello,i’m sorry to bother you again,there is a problem that has puzzled me for a long time. I really can't figure out where to correct it.I trained with one  gpu. I made the following modifications to the code(train.py):
![image](https://user-images.githubusercontent.com/59550614/199511227-95a2b372-30fe-470f-9f07-6d440f47666e.png)
and
![image](https://user-images.githubusercontent.com/59550614/199511513-527c7951-08db-4b3c-ad9e-0630ae1771f9.png)
and
![image](https://user-images.githubusercontent.com/59550614/199511675-f5fbc2c0-c0a4-4bf5-8250-6b4707fe62a4.png)
and
![image](https://user-images.githubusercontent.com/59550614/199511926-24e3899c-6e97-40f7-8d12-cd03173c8512.png)
The problem is that my training results are poor.I don't know what the problem is.

**Comment by yjq767579182** [comment]
> 打扰一下，我想知道如何只用一个GPU进行训练，你能帮我吗？多谢。
你好 请问你解决了吗 我也只有一个GPU可用 需要怎么改呢

**Comment by XieZhiyao21** [comment]
do you solve this problem?i want to know how to deal with that too.
